dockers/docker-database/flush_unused_database

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+import swsssdk
+import redis
+import subprocess
+import time
+
+output = subprocess.check_output('sonic-db-cli PING', stderr=subprocess.STDOUT, shell=True)
+while('PONG' not in output):
+    time.sleep(1)
```

Comment:
If PONG is not in output, you will sleep forever.

Reply:
updated

Reply:
It is improved, but you can improve even more:
1. if any output is good, no need to sleep and you can immediately exit loop
2. no need to check output at the beginning.

---
In reply to: [536364747](https://github.com/Azure/sonic-buildimage/pull/5773#discussion_r536364747) [](ancestors = 536364747)

Reply:
It should be OK now